rust/xaynet-server/src/storage/tests/mod.rs

```diff
@@ -132,3 +142,7 @@ pub async fn write_local_seed_entries(
 
     update_result
 }
+
+pub fn create_global_model(byte_size: usize) -> Model {
```

Comment:
`byte_size` is quite misleading, since the `Model::from_primitives()` will cast the `Vec<i32/64>` into a `Vec<Ratio<BigInt>>` where the byte size depends on the chosen mask config. maybe `len` or `nb_elements` is a better name.